### PR TITLE
NIFI-1788 Fixed listed allowable values

### PR DESCRIPTION
**Pull request by pvillard31**
- fixed list of allowable values
- replaced related deprecated method when writing

### CODE REVIEW

File: nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/SequenceFileWriterImpl.java [hunk 1]
```diff
@@ -94,8 +94,10 @@ public void process(InputStream in, OutputStream out) throws IOException {
 
                 try (final FSDataOutputStream fsDataOutputStream = new FSDataOutputStream(bwos, new Statistics(""));
                         final SequenceFile.Writer writer = SequenceFile.createWriter(configuration,
-                                fsDataOutputStream, Text.class, InputStreamWritable.class, compressionType,
-                                new DefaultCodec())) {
+                                SequenceFile.Writer.stream(fsDataOutputStream),
+                                SequenceFile.Writer.keyClass(Text.class),
+                                SequenceFile.Writer.valueClass(InputStreamWritable.class),
+                                SequenceFile.Writer.compression(compressionType, new DefaultCodec()))) {
```
Comment (pvillard31) [inline nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/SequenceFileWriterImpl.java]: Is it expected to use DefaultCodec? I saw we have a property for compression codec, shouldn't we propose something to use this property at this moment?

Comment (olegz) [inline nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/SequenceFileWriterImpl.java]: @pvillard31 I would agree. In any event raising an improvement JIRA will let such discussion to begin.

**Comment by pvillard31** [comment]
@olegz You are absolutely right! Thanks for the lines of code, added the test in an existing test class for this.

File: nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/SequenceFileWriterImpl.java (hunk 1 above)
Comment (pvillard31) [inline nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/SequenceFileWriterImpl.java]: @olegz OK, thanks, I'll raise a JIRA shortly

**Pull request closed without merging**